docs/source/Tutorial.md

```diff
@@ -892,6 +1015,39 @@ a bit more flexibility.
   auto orc = monster_builder.Finish();
 ~~~
 </div>
+<div class="language-c">
+<br>
+*Note: Since we are passing `150` as the `mana` field, which happens to be the
```

Comment:
Ok, that's weird, since the comment holds for all languages. Feel free to move it outside the C++ div. Though the general principle still holds, we want to keep per-language segments as small and as uniform as possible, for maintainability